dist/manifest.json

```diff
@@ -33,7 +33,10 @@
       "https://superuser.com/*",
       "https://askubuntu.com/*",
       "http://stackoverflow.com/*",
-      "https://stackoverflow.com/*"
+      "https://stackoverflow.com/*",
+      "https://tableless.com.br/*",
+      "https://code.tutsplus.com/*",
+      "https://laracasts.com/discuss/*"
```

Comment:
@JeffreyWay looks like laracasts users are interested in having "copy to clipboard" functionality, maybe it's a good opportunity to add it ;)